turbopack/crates/turbopack-core/src/resolve/mod.rs

```diff
@@ -1287,6 +1291,9 @@ async fn find_package(
                     }
                 }
                 for extension in &options.extensions {
+                    if excluded_extensions.contains(&Vc::cell(extension.to_owned())) {
```

Comment:
This clone and cell juggling in the for loop seem bad for performance. But I couldn't find another way to make `Vc<IndexSet<_>>` work.